go-controller/pkg/clustermanager/pod/allocator_test.go

```diff
@@ -637,6 +639,7 @@ func TestPodAllocator_reconcileForNAD(t *testing.T) {
 
 			err = a.reconcile(old, new, tt.args.release)
 			if len(tt.expectError) > 0 {
+				fmt.Printf("tt.name: %s\n", tt.name)
 				g.Expect(err).To(gomega.MatchError(gomega.ContainSubstring(tt.expectError)))
```

Comment:
```suggestion
g.Expect(err).To(gomega.MatchError(gomega.ContainSubstring(tt.expectError)), fmt.Sprintf("tt.name: %s", tt.name))
```

Not sure it's worth it.